src/args/agnostic.nix

```diff
@@ -55,6 +55,7 @@
     fromToml = builtins.fromTOML;
     fromYaml = import ./from-yaml/default.nix self;
     fromYamlFile = path: self.fromYaml (builtins.readFile path);
+    getAllPkgsFromToml = import ./get-all-pkgs-from-toml/default.nix self;
```

Comment:
Too specific to be an agnostic arg, simply define an extra file within `make-python-environment`

Reply:
better yet, simply do it within the `let` of `make-python-environment/default.nix`